### PR TITLE
Add CVE-2016-5385 for Humbug/file_get_contents

### PR DESCRIPTION
**Pull request by theofidry**
/cc @kelunik

### CODE REVIEW

File: padraic/file_get_contents/CVE-2016-5385.yaml [hunk 1]
```diff
@@ -0,0 +1,7 @@
+title:     HTTP Proxy header vulnerability
+link:      https://github.com/humbug/file_get_contents/releases/tag/1.1.2
+cve:       CVE-2016-5385
+branches:
```
Comment (Ocramius) [inline padraic/file_get_contents/CVE-2016-5385.yaml]: The `branches` entry seems invalid here, as we are missing a branch name reference

Comment (fabpot) [reply padraic/file_get_contents/CVE-2016-5385.yaml]: indeed :(

Comment (Ocramius) [reply padraic/file_get_contents/CVE-2016-5385.yaml]: I'll send a patch in a few moments :-)

Comment (Ocramius) [reply padraic/file_get_contents/CVE-2016-5385.yaml]: https://github.com/FriendsOfPHP/security-advisories/pull/252

Comment (theofidry) [reply padraic/file_get_contents/CVE-2016-5385.yaml]: Ah damn, sorry about that looks like I deleted the line by accident. Thanks Marco for the patch

**Pull request merged**